bitbots_localization/cfg/localization.cfg

```diff
@@ -60,18 +60,12 @@ group_visualization.add("debug_visualization", bool_t, 0, "publish debug visuali
 # filter stuff
 group_pf.add("particle_number", int_t, 0, "The amount of particles", None, min=1, max=10000)
 group_pf.add("resampling_interval", int_t, 0, "Number of filterupdates performed before resampling", None, min=0, max=10000)
-group_pf.add("use_lines", bool_t, 0, "use line information", None)
-group_pf.add("use_goals", bool_t, 0, "use goal information", None)
-group_pf.add("use_fieldboundary", bool_t, 0, "use fieldboundary information", None)
-group_pf.add("use_corners", bool_t, 0, "use corners information", None)
-group_pf.add("use_tcrossings", bool_t, 0, "use tcrossing information", None)
-group_pf.add("use_crosses", bool_t, 0, "use crosses information", None)
 group_pf.add("min_weight", double_t, 0, "The minimal weight of a particle", None, min=0.0, max=1)
 group_pf.add("min_resampling_weight", double_t, 0, "The minimal weight of the best particle to resample", None, min=0.0, max=1)
 group_pf.add("percentage_best_particles", int_t, 0, "Percentage of best particles to generate pose from", None, min=0, max=100)
 group_pf.add("distance_factor", double_t, 0, "how much distant measurements get their weights reduced", None, min=0.0, max=1)
 group_pf.add("lines_factor", double_t, 0, "how much line measurements are contribute to the particle weight", None, min=0.0, max=1)
-group_pf.add("goals_factor", double_t, 0, "how much goal measurements are contribute to the particle weight", None, min=0.0, max=1)
+group_pf.add("goal_factor", double_t, 0, "how much goal measurements are contribute to the particle weight", None, min=0.0, max=1)
```

Comment:
why that? all other things that appear more than once on the field are also in plural here

Reply:
Yes it seems better, but until now goal was used in the coder frequently, while the parameter was called goals. So I switched it to the more common usage in the code.